x-pack/legacy/plugins/siem/public/pages/case/components/case_view/index.tsx

```diff
@@ -160,66 +129,8 @@ export const CaseComponent = React.memo<CaseProps>(({ caseId, initialData, isLoa
       onClick: () => null,
     },
   ];
-  const userActions = [
-    {
-      avatarName: data.createdBy.username,
-      title: (
-        <EuiFlexGroup alignItems="baseline" gutterSize="none" justifyContent="spaceBetween">
-          <EuiFlexItem grow={false}>
-            <p>
-              <strong>{`${data.createdBy.username}`}</strong>
-              {` ${i18n.ADDED_DESCRIPTION} `}{' '}
-              <FormattedRelativePreferenceDate value={data.createdAt} />
-              {/* STEPH FIX come back and add label `on` */}
-            </p>
-          </EuiFlexItem>
-          <EuiFlexItem grow={false}>
-            <PropertyActions propertyActions={propertyActions} />
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      ),
-      children: isEditDescription ? (
-        <>
-          <DescriptionMarkdown
-            descriptionInputHeight={200}
-            initialDescription={data.description}
-            isLoading={isLoading}
-            onChange={updatedDescription => setDescription(updatedDescription)}
-          />
-
-          <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false} wrap={true}>
-            <EuiFlexItem grow={false}>
-              <EuiButton
-                fill
-                isDisabled={isLoading}
-                isLoading={isLoading}
-                onClick={() => onUpdateField('description', description)}
-              >
-                {i18n.SUBMIT}
-              </EuiButton>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButtonEmpty onClick={() => setIsEditDescription(false)}>
-                {i18n.CANCEL}
-              </EuiButtonEmpty>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </>
-      ) : (
-        <Markdown raw={data.description} data-test-subj="case-view-description" />
-      ),
-    },
-  ];
 
-  const onSubmit = useCallback(
-    newTitle => {
-      onUpdateField('title', newTitle);
-      setTitle(newTitle);
-    },
-    [title]
-  );
-
-  const titleNode = <EditableTitle isLoading={isLoading} title={title} onSubmit={onSubmit} />;
+  const onSubmit = useCallback(newTitle => onUpdateField('title', newTitle), [data.title]);
```

Comment:
you should have a dependency on `onUpdateField`
```suggestion
  const onSubmit = useCallback(newTitle => onUpdateField('title', newTitle), [onUpdateField]);
```